buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -358,6 +375,34 @@ class ClusterFormationTasks {
         return copyPlugins
     }
 
+    /** Configures task to copy a plugin based on a zip file resolved using dependencies for an older version */
+    static Task configureCopyBwcPluginsTask(String name, Project project, Task setup, NodeInfo node) {
+        for (Map.Entry<String, Project> plugin : node.config.plugins.entrySet()) {
+            System.err.println(plugin.getKey() + ": " + plugin.getValue())
+            Project pluginProject = plugin.getValue()
+            if (pluginProject.plugins.hasPlugin(PluginBuildPlugin) == false) {
```

Comment:
the same check is also used in `#configureBwcPluginDependency()`. Does it make sense to encapsulate it? Also the exception message is slightly inconsistent.